mock/mock.spec

```diff
@@ -188,6 +194,7 @@ cp -a etc/pam/* %{buildroot}%{_sysconfdir}/pam.d/
 
 install -d %{buildroot}%{_sysconfdir}/mock
 cp -a etc/mock/* %{buildroot}%{_sysconfdir}/mock/
+cp -a etc/isolated-build.cfg %{buildroot}%{_sysconfdir}/mock/
```

Comment:
Can you put `isolated-build.cfg` to 'etc/mock' ? It will save several special handling like this one.